test/index.js

```diff
@@ -214,6 +214,22 @@ describe('locale-specific pluralization rules', function () {
     expect(polyglot.t('n_votes', 102)).to.equal('102 صوت');
   });
 
+  it('interpolates properly in Arabic', function () {
+    var phrases = {
+      hello: 'الرمز ${code} غير صحيح' // eslint-disable-line no-template-curly-in-string
+    };
+
+    var polyglot = new Polyglot({
+      phrases: phrases,
+      locale: 'ar',
+      interpolation: { prefix: '${', suffix: '}' }
+    });
+
+    expect(polyglot.t('hello', { code: 'De30Niro' })).to.equal('الرمز De30Niro غير صحيح');
+    expect(polyglot.t('hello', { code: '30DeNiro' })).to.equal('الرمز 30DeNiro غير صحيح');
```

Comment:
I don't know enough about this to be certain that this assertion is correct. Is there something I could reference?

Reply:
The linked issue has a bunch of relevant discussion.

Reply:
Ah, I see, thank you! Can you please add a code comment that explains that this can't be easily handled in polyglot and include a link to the discussion?

Reply:
sure, updated